### PR TITLE
ER-580 disability confidence application review

### PR DESCRIPTION
**Pull request by chrisjensenuk**
Showing an extra column with the disability status in the application reviews for Disability Confident applications.

### CODE REVIEW

File: src/Employer/Employer.Web/Views/ApplicationReview/ApplicationReview.cshtml [hunk 1]
```diff
@@ -2,6 +2,15 @@
 @using Esfa.Recruit.Vacancies.Client.Domain.Entities
 @model Esfa.Recruit.Employer.Web.ViewModels.ApplicationReview.ApplicationReviewViewModel
 
+<div asp-condition="@Model.ShowDisabilityStatusAlert" class="grid-row">
+    <div class="column-full">
+        <div class="error-summary small-top-margin" role="alert" aria-labelledby="error-summary-heading-example-2" tabindex="-1" style="border:5px solid #2e358b;">
```
Comment (sachpatel) [inline src/Employer/Employer.Web/Views/ApplicationReview/ApplicationReview.cshtml]: There should already be a css class for this border I think.

File: src/Employer/Employer.Web/Views/Shared/_VacancySummaryDisplayPartial.cshtml [hunk 2]
```diff
@@ -22,7 +22,8 @@
     </div>
     <div asp-condition="Model.IsDisabilityConfident" class="grid-row" >
         <div class="column-two-thirds small-bottom-margin">
-            <img src="/img/disability-confident_545x172.png" class="disability-confident-logo" alt="Employer is registered as disability confident">
+            <img src="/img/disability-confident_545x172.png" class="disability-confident-logo" alt="A logo confirming that the employer is registered as disability confident">
+            <span class="visuallyhidden">This vacancy meets disability confident requirements</span>
```
Comment (sachpatel) [inline src/Employer/Employer.Web/Views/Shared/_VacancySummaryDisplayPartial.cshtml]: Does this visually hidden classed element not have to appear on the preview pages as well?

File: src/Employer/Employer.Web/Views/ApplicationReview/ApplicationReview.cshtml (hunk 1 above)
Comment (chrisjensenuk) [reply src/Employer/Employer.Web/Views/ApplicationReview/ApplicationReview.cshtml]: 👍

File: src/Employer/Employer.Web/Views/Shared/_VacancySummaryDisplayPartial.cshtml (hunk 2 above)
Comment (chrisjensenuk) [reply src/Employer/Employer.Web/Views/Shared/_VacancySummaryDisplayPartial.cshtml]: 👍

**Pull request merged**